schemas/src/main/java/com/scottlogic/deg/schemas/v0_1/ProfileSchemaValidatorLeadPony.java

```diff
@@ -0,0 +1,76 @@
+package com.scottlogic.deg.schemas.v0_1;
+
+import com.scottlogic.deg.schemas.common.ValidationResult;
+import org.leadpony.justify.api.JsonSchema;
+import org.leadpony.justify.api.JsonValidationService;
+import org.leadpony.justify.api.ProblemHandler;
+
+import javax.json.stream.JsonParser;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Used to validate a DataHelix Profile JSON file.
+ * <p>
+ * Checks that the profile JSON file is valid against the DataHelix Profile Schema (datahelix.schema.json)
+ */
+public class ProfileSchemaValidatorLeadPony implements ProfileSchemaValidator {
+
+    @Override
+    public ValidationResult validateProfile(File profileFile) {
+        try {
+            return validateProfile(new FileInputStream(profileFile));
+        } catch (FileNotFoundException e) {
+            List<String> errMsgs = new ArrayList<>();
+            errMsgs.add(e.getLocalizedMessage());
+            return new ValidationResult(errMsgs);
+        }
+    }
+
+    /**
+     * Validates a json file against the DataHelix Profile JSON Schema.
+     *
+     * @return the result of validating the provided profile
+     */
+    @Override
+    public ValidationResult validateProfile(InputStream profileStream) {
+        ValidationResult result = null;
+        result = validateProfile(this.getClass().getResourceAsStream(datahelixProfileSchema), profileStream);
+        return result;
```

Comment:
is the `null` initialisation required? Can't you just directly return `validateProfile(this.getClass().getResourceAsStream(datahelixProfileSchema), profileStream)`?